.github/workflows/regen_certs.yml

```diff
@@ -0,0 +1,35 @@
+name: Regenerate WPT certificates
+on:
+  schedule:
+    # Our certificates are good for ~2 years, but we regenerate them every
```

Comment:
I think two years is a problem; 398 days seems to be the maximum for at least Safari going forward.

Reply:
```suggestion
    # Our certificates are good for ~1 year, but we regenerate them every
```

Yeah I've fixed the config in another PR, so this is just outdated comment.